insights/parsers/localectl.py

```diff
@@ -0,0 +1,42 @@
+"""
+Localectl - command ``localectl``
+=================================
+Parser the output of localectl command.
+
+"""
+
+from insights.core import CommandParser
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.localectl)
+class Localectl(CommandParser, dict):
```

Comment:
How about renaming the parser name as `LocaleCtlStatus` with CamelCase Style and adding the `Status`?

Reply:
UPdate